apps/researcher/cypress/e2e/object.cy.ts

```diff
@@ -0,0 +1,26 @@
+describe('Object details page', () => {
+  it('loads an existing object', () => {
+    cy.visit('/en');
+    // Get the name of the first object in the list.
+    cy.getBySel('object-card-name')
+      .first()
+      .then($cardName => {
+        // Navigate to the first object details page.
+        cy.getBySel('object-card-name').first().click();
+        // Wait for the page to load.
+        cy.location('pathname', {timeout: 60000}).should('include', '/object');
+        // On the details page.
+        cy.getBySel('error').should('not.exist');
+        cy.getBySel('page-title').then($detailsName => {
+          expect($cardName.text()).equal($detailsName.text());
+          cy.getBySel('no-object').should('not.exist');
+        });
+      });
+  });
+
+  it('shows an error message if no object can be found', () => {
```

Comment:
'no object', I think, doesn't make clear that a specific object is requested. Perhaps: `shows an error message if no object matches the ID` or `shows an error message if the object is not found`?